generate/generate_hcl_test.go

```diff
@@ -958,7 +959,7 @@ func TestWontOverwriteManuallyDefinedTerraform(t *testing.T) {
 		fmt.Sprintf("f:stack/%s:%s", genFilename, manualTfCode),
 	})
 
-	report := generate.Do(s.Config(), s.RootDir())
+	report := generate.Do(s.Config(), s.RootDir(), project.NewPath("/modules"), nil)
```

Comment:
we have this several times, maybe it could be a:
```
const TestVendorDir project.Path = "/modules"
```
inside the `generate_test.go`.